src/Microsoft.DocAsCode.Build.Engine/CompilePhaseHandlerWithIncremental.cs

```diff
@@ -190,12 +190,9 @@ private IEnumerable<string> GetFilesToRelayMessages(HostService hs)
 
                 // warnings from token file won't be delegated to article, so we need to add it manually
                 var key = ((RelativePath)f).GetPathFromWorkingFolder();
-                foreach (var item in CurrentBuildVersionInfo.Dependency.GetAllDependencyFrom(key))
+                foreach (var item in CurrentBuildVersionInfo.Dependency.GetAllIncludeDependencyFrom(key))
                 {
-                    if (item.Type == DependencyTypeName.Include)
-                    {
-                        files.Add(((RelativePath)item.To.Value).RemoveWorkingFolder());
-                    }
+                    files.Add(((RelativePath)item).RemoveWorkingFolder());
```

Comment:
discussed offline, includ dependency is currently for relative, we can keep for now.

---
In reply to: [108099254](https://github.com/dotnet/docfx/pull/1492#discussion_r108099254) [](ancestors = 108099254)